server/src/main/java/com/tencent/rss/server/ShuffleServerMetrics.java

```diff
@@ -53,33 +53,33 @@
   private static final String TOTAL_UPLOAD_TIME_S = "total_upload_time_s";
   private static final String TOTAL_DROPPED_EVENT_NUM = "total_dropped_event_num";
 
-  static Counter counterTotalReceivedDataSize;
-  static Counter counterTotalWriteDataSize;
-  static Counter counterTotalWriteBlockSize;
-  static Counter counterTotalWriteTime;
-  static Counter counterWriteException;
-  static Counter counterWriteSlow;
-  static Counter counterWriteTotal;
-  static Counter counterEventSizeThresholdLevel1;
-  static Counter counterEventSizeThresholdLevel2;
-  static Counter counterEventSizeThresholdLevel3;
-  static Counter counterEventSizeThresholdLevel4;
-  static Counter counterTotalReadDataSize;
-  static Counter counterTotalReadTime;
-  static Counter counterTotalUploadSize;
-  static Counter counterTotalUploadTimeS;
-  static Counter counterTotalDroppedEventNum;
+  public static Counter counterTotalReceivedDataSize;
```

Comment:
why do we make these variable public?

Reply:
new package is created for buffer related class, these variable have to be public to access